docs/docs/extend/plugins/label.md

```diff
@@ -63,6 +63,32 @@ If the `print_labels` method is not changed, this will run the `print_label` met
 !!! tip "Custom Code"
     If your plugin overrides the `print_labels` method, you will have to ensure that the label printing is correctly offloaded to the background worker. Look at the `offload_label` method of the plugin mixin class for how this can be achieved.
 
+### Printing options
+
+A printing plugin can define custom options that get shown on the printing screen and get passed to the `print_labels`/`print_label` function as a kwarg called `printing_options`. This can be used to e.g. let the user dynamically select the orientation of the label, the color mode, ... for each print job.
+The following simple example shows how to implement an orientation select. For more information about how to define fields, refer to the django rest framework (DRF) [documentation](https://www.django-rest-framework.org/api-guide/fields/).
+
+```py
+from rest_framework import serializers
+
+class MyLabelPrinter(LabelPrintingMixin, InvenTreePlugin):
+    ...
+
+    class PrintingOptionsSerializer(serializers.Serializer):
```

Comment:
Though this would be enough, but I'll add another specific note.

> If your plugin wants to dynamically expose options based on the request, you can implement the get_printing_options_serializer function which by default returns an instance of the PrintingOptionsSerializer class if defined.

Reply:
Do you think how I changed it is enough?

Reply:
Yes, seems more clear this way IMO